dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -32,9 +32,19 @@ RUN apt-get update &&       \
 RUN pip install \
         scapy==2.4.2 \
         setuptools \
-        pyroute2==0.5.3 \
+        pyroute2==0.5.3
+{% if CONFIGURED_ARCH == "armhf" %}
+## Fix for gcc/python not found in arm docker
+RUN apt-get install -y gcc-6
+RUN ln -s -f /usr/bin/gcc-6 /usr/bin/arm-linux-gnueabihf-gcc
+RUN apt-get install -f -y python2.7 python2.7-dev
```

Comment:
why do we need gcc-6 here?

why python2.7 is not available in arm docker?

if you need python2.7, can you move apt-get install before line 32?

Reply:
when pip compiles netifaces, it got failed in arm.
As requested moved the pkg installation above at line 32